Kernel/Scheduler.h

```diff
@@ -32,6 +33,7 @@ class Scheduler {
     static void beep();
     static void idle_loop();
     static void stop_idling();
+    static void add_timer(Timer& timer);
```

Comment:
No need to name the argument, since the name is embedded in the type:
```suggestion
    static void add_timer(Timer&);
```